tests/bcr/pkg/pkg_test.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/DataDog/sketches-go/ddsketch"
 	"github.com/bazelbuild/bazel-gazelle/tests/bcr/pkg/data"
+	"github.com/bazelbuild/buildtools/labels"
```

Comment:
You can temporarily fix the CI failure with a `gazelle_override` for `build_naming_convention`. If https://github.com/bazelbuild/buildtools/pull/1173 isn't merged soon, we could think about adding this as a default override. What do you think?

Reply:
Yep, that works. I wanted to come up with some way to differentiate an `archive` from another dependency, to show that the test can only work with the archive.